.github/workflows/test.yml

```diff
@@ -0,0 +1,85 @@
+name: Run All Tests
+
+on:
+  pull_request:
+    types: [
+      opened,
+      labeled,
+      ready_for_review,
+      reopened,
+      synchronize,
+    ]
+    branches:
+      - main
+      - release**
+      - production**
+      - hotfix**
+      - lab-**
+
+env:
+  NODE_VERSION: '14.x'
+
+jobs:
+  Type:
```

Comment:
the job name `Type` here looks a bit confusing (despite of the workflow name is test), shall we name it `Type-Test`, `Lint-Test`, or something?